dev/playbooks/chd-setup.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Can you add a yaml comment on the top to explain the purpose of this? this will help new users.

Reply:
done, thanks for the suggestion!